src/LibraryWindow.vala

```diff
@@ -298,6 +298,19 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
             }
         });
 
+        library_manager.search_finished.connect (() => {
+            print ("search endedn\n");
```

Comment:
Probably want to either change this to a `debug ()` or remove it before merging :)

Reply:
Oops, of course. :grin: